plugins/slack_app/app/helpers/slack_app_helper.rb

```diff
@@ -0,0 +1,13 @@
+require 'uri'
+
+module SlackAppHelper
+  def oauth_url(scopes)
```

Comment:
this needs to be renamed ... helpers are all included into the view -> global -> slack_oauth_url